src/components/app/details/main.tsx

```diff
@@ -72,131 +72,140 @@ export default function AppDetailsPage({ isV2 }) {
                 </Suspense>
             </ErrorBoundary>
         </div>
-    );
+    )
 }
 
 export function AppHeader() {
-    const { appId } = useParams<{ appId }>();
-    const match = useRouteMatch();
-    const history = useHistory();
-    const location = useLocation();
-    const currentPathname = useRef('');
-    const [showInfoModal, setShowInfoModal] = useState(false);
-    const [submitting, setSubmitting] = useState(false);
+    const { appId } = useParams<{ appId }>()
+    const match = useRouteMatch()
+    const history = useHistory()
+    const location = useLocation()
+    const currentPathname = useRef('')
+    const [showInfoModal, setShowInfoModal] = useState(false)
+    const [submitting, setSubmitting] = useState(false)
     const [labelTags, setLabelTags] = useState<{ tags: OptionType[]; inputTagValue: string; tagError: string }>({
         tags: [],
         inputTagValue: '',
         tagError: '',
-    });
-    const [result, setResult] = useState(undefined);
-    const [isLoading, setIsLoading] = useState(false);
-    const [appName, setAppName] = useState('');
+    })
+    const [result, setResult] = useState(undefined)
+    const [isLoading, setIsLoading] = useState(false)
+    const [appName, setAppName] = useState('')
 
     const getAppMetaInfoRes = () => {
-        setIsLoading(true);
+        setIsLoading(true)
         const res = getAppMetaInfo(appId).then((_result) => {
-            setAppName(_result?.result?.appName);
+            setAppName(_result?.result?.appName)
             let labelOptionRes = _result?.result?.labels?.map((_label) => {
                 return {
                     label: `${_label.key?.toString()}:${_label.value?.toString()}`,
                     value: `${_label.key?.toString()}:${_label.value?.toString()}`,
-                };
-            });
-            setResult(_result);
-            setIsLoading(false);
-            setLabelTags({ tags: labelOptionRes || [], inputTagValue: '', tagError: '' });
-        });
-    };
+                }
+            })
+            setResult(_result)
+            setIsLoading(false)
+            setLabelTags({ tags: labelOptionRes || [], inputTagValue: '', tagError: '' })
+        })
+    }
 
     useEffect(() => {
+        let isMounted = true
```

Comment:
do  we need this variable? please remove commented code

Reply:
Will remove, that is not needed